utils/objects.py

```diff
@@ -15,17 +15,17 @@ def __init__(self, events):
 
     def goodElectrons(self):
         # # Good Electrons
-        electronQualityCut = (self.electrons.pt > self.leptonPt) & (abs(self.electrons.eta) < self.etaCut) & (self.electrons.iso < 0.1)
+        electronQualityCut = (self.electrons.pt > self.leptonPt) & (abs(self.electrons.eta) < self.etaCut)
         return self.electrons[electronQualityCut]
 
     def goodMuons(self):
         # # Good Muons
-        muonQualityCut = (self.muons.pt > self.leptonPt) & (abs(self.muons.eta) < self.etaCut) & (self.muons.iso < 0.4)
+        muonQualityCut = (self.muons.pt > self.leptonPt) & (abs(self.muons.eta) < self.etaCut)
         return self.muons[muonQualityCut]
 
     def goodJets(self):
         # # Good AK4 Jets Cut
-        ak4QualityCut = (self.jets.pt > 30) & (abs(self.jets.eta) < 2.4) & (self.jets.ID == True)
+        ak4QualityCut = (self.jets.pt > 30) & (abs(self.jets.eta) < 5.0) & (self.jets.ID == True)
```

Comment:
We should keep the eta for the AK4 jets at 2.4. I think the confusion was what we do for the AK8 jets.